benchmarks/templates/benchmarks/benchmark.html

```diff
@@ -0,0 +1,181 @@
+{% extends "benchmarks/components/app-view.html" %}
+{% load static %}
+
+{% block banner %}
+    <h1 class="title">{{ benchmark.benchmark_type.identifier }}</h1>
+    <p>
+        {# Paper #}
+        {% if reference_identifier %}
+            <button class="button">
+                <a href="{{ reference_url }}" target="_blank">
+                    <i class="fas fa-file-alt"></i> {{ reference_identifier }}
+                </a>
+            </button>
+        {% else %}
+            <button class="button disabled">
+                No reference provided
+            </button>
+        {% endif %}
+
+        {# Code #}
+        <button class="button">
+            <a href="https://github.com/search?q=repo%3Abrain-score%2F{{ benchmark.benchmark_type.domain }}%20{{ benchmark.benchmark_type.identifier }}&type=code"
+               target="_blank">
+                <i class="fa-brands fa-github"></i> Find on GitHub
+            </a>
+        </button>
+    </p>
+{% endblock %}
+
+{# Right-hand info #}
+{% block info_section %}
+    <div class="box">
+        <div class="card-content">
+            <p class="subtitle is-5">
+                Ceiling
+            </p>
+            {% if benchmark.ceiling %}
+                <span class="has-text-weight-bold">{{ benchmark.ceiling|floatformat:2 }}</span>.
+                <p>Note that scores are relative to this ceiling.</p>
+            {% else %}
+                Not available
+            {% endif %}
+        </div>
+    </div>
+
+    <div class="box">
+        <div class="card-content">
+            <p class="subtitle is-5">
+                Data: {{ data_identifier }}
+            </p>
+            <span>
+                {# Stimuli #}
+                {% if benchmark.meta and benchmark.meta.number_of_stimuli %}
+                    {{ benchmark.meta.number_of_stimuli }} stimuli
+                {% endif %}
+
+                {# Brain/behavior data #}
+                {% if benchmark.meta and benchmark.meta.number_of_recording_sites %}
+                    recordings from
+                    {{ benchmark.meta.number_of_recording_sites }}
+                    sites in
+                    {{ benchmark.meta.recording_sites }}
+                {% endif %}
+                {% if benchmark.meta and benchmark.meta.behavioral_task %}
+                    {{ benchmark.meta.behavioral_task }} task
+                {% endif %}
+            </span>
+            <button class="button">
+                <a href="https://github.com/search?q=repo%3Abrain-score%2F{{ benchmark.benchmark_type.domain }}%20{{ data_identifier }}&type=code"
+                   target="_blank">
+                    <i class="fa-brands fa-github"></i> Find on GitHub
+                </a>
+            </button>
```

Comment:
Button is inline with text. Not as aesthetic. Could use <br> or `position: block`

Reply:
Will address in a subsequent PR.